app/redux/redux-actions/adminActions.js

```diff
@@ -104,7 +104,7 @@ export const reduxFetchInitialContent = (auth) => (dispatch) => {
     apiCall(isSuperAdmin ? "/tasks.functions.list" : "/tasks.functions.list"),
```

Comment:
Question: @abdullai-t : these lines don't seem to make sense.  Should non-super admins be fetching the task lists and functions?

Reply:
I don't we should be fetching tasks and task function for Cadmins. I add a check for that in my last commit